node/src/chain_spec/dev.rs

```diff
@@ -72,7 +70,7 @@ pub fn dev_config(
                     crowdloan_fund_pot: zeitgeist_primitives::constants::BASE.saturating_mul(100),
                     inflation_info: crate::chain_spec::DEFAULT_COLLATOR_INFLATION_INFO,
                     nominations: vec![],
-                    parachain_id,
+                    parachain_id: 2050_u32.into(),
```

Comment:
In `dev_config` I am pretty sure it was `2101` before, but I don't think this is relevant, since it's only used for a local development environment.